modules/backend/widgets/form/assets/js/october.form.js

```diff
@@ -39,9 +39,15 @@
 
         this.$el.on('oc.triggerOn.afterUpdate', this.proxy(this.toggleEmptyTabs))
         this.$el.one('dispose-control', this.proxy(this.dispose))
+        this.$form.closest('.modal').on('hidden.oc.popup', this.proxy(this.dispose))
```

Comment:
If you have recordfinder field type in relation popup and have some fields that depends on it, than this line causes to break dependsOn functionality. If you select something from recordfinder it will not refresh other fields.

Reply:
Thanks for letting me know. It seems like we need to bind the hidden popup event on a different form element. I'll look into this.